docs/integrations/prisma.md

```diff
@@ -0,0 +1,221 @@
+# Integrating Prisma with Mercurius
+
+[Prisma](https://prisma.io) is an [open-source](https://github.com/prisma/prisma) ORM for Node.js and TypeScript. 
+It can be used as an _alternative_ to writing plain SQL, or using another database access tool such as SQL query builders (e.g. [knex.js](https://knexjs.org/)) or ORMs (like [TypeORM](https://typeorm.io/) and [Sequelize](https://sequelize.org/)).
+Prisma currently supports PostgreSQL, MySQL, SQL Server and SQLite.
+
+You can easily combine Prisma and Mercurius to build your GraphQL server that connects to a database. Prisma is agnostic to the GraphQL tools you use when building your GraphQL server. 
+
+Prisma can be used with plain JavaScript and it embraces TypeScript and provides a level to type-safety that goes beyond the guarantees other ORMs in the TypeScript ecosystem. You can find an in-depth comparison of Prisma against other ORMs [here](https://www.prisma.io/docs/concepts/more/comparisons)
+
+## Installation
+
+Install [Prisma CLI](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-cli) as a development dependency in your project:
+
+```bash
+npm install prisma --save-dev
+npm install @prisma/client
```

Comment:
Can you please include the major version? So we know at least which version this was tested on.

Reply:
We don't follow semantic versioning, i.e., we introduce breaking changes in minor versions. Prisma is currently at v2.20.1, and we have minor releases every 2 weeks.

Reply:
Then include a full version number here, as we would know which version you tested this in :).

Reply:
Here's more details on how we do our [versioning](https://www.prisma.io/docs/about/releases#versioning)